mayastor/tests/block_device_nvmf.rs

```diff
@@ -885,3 +911,167 @@ async fn nvmf_device_reset() {
     })
     .await;
 }
+
+async fn wipe_device_blocks(is_unmap: bool) {
+    let ms = get_ms();
+    let url = launch_instance().await;
+    let url2 = url.clone();
+
+    struct DeviceIoCtx {
+        handle: Box<dyn BlockDeviceHandle>,
+    }
+
+    const BUF_SIZE: u64 = 32768;
+    const OP_OFFSET: u64 = 12 * 1024 * 1024;
+
+    // Read completion callback.
+    fn wipe_completion_callback(success: bool, ctx: *const c_void) {
+        // Make sure callback is invoked only once.
+        flag_callback_invocation();
+
+        assert!(success, "block deallocation failed");
+        // Make sure we were passed tha same pattern string as requested.
```

Comment:
typo: "tha" -> "the"

Reply:
Thanks, fixed ! (also in 4 other places :-)